levels/level-4.md

```diff
@@ -0,0 +1,27 @@
+# Brukergrensesnittutvikler, nivå 4
+
+## Tittel
+
+Sjefsbrukergrensesnittutvikler / Lead User Interface Developer
+
+## Beskrivelse
+
+Det forventes at den ansatte er en tydelig fagperson både internt og eksternt og deler fagkunnskap gjennom presentasjoner, workshops o.l utafor selskapet, f.eks. på meetups og konferanser.
+
+## Erfaring
+
+8 år eller lengre erfaring i en relevant stilling
+
+## Faglig kompetanse
+
+Veldig god forståelse for med oppsett og tilpasning av verktøy og systemer i bruk i produksjonsprosessen
+
+Veldig god forståelse for HTTP
```

Comment:
I think perhaps as a sjefsbrukergrensesnittsutvikler all the other things you do are more important than any number of really, really good understanding of http.